### ISSUE TITLE
Prettier is flattening ternary operators

### ISSUE THREAD

**Issue by mattmazzola** [prettier/prettier#3911]
**Prettier 1.10.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACSKAZ33hLwF48AKNJPE7ASygHM8AfPKAVwFsARnGwBKSgD4AOlDx4YATwxwIAMzxpKFKgHJGLVtumSYAfjwYAhtiJwAkrFpiA1HgCMABiMx6GlwCZ3EAAaEAgMGCZoImRQK2wIAHcABSsEaJQLADcIJgATYJALEmQVCwAbGxCBbAswAGs4GABlSzB9ZBhsHjgQ3IgwEvLKkBYbXCSa1j4LQYqekAArIjQAIRr6xqaLPjgAGRY4WeGIHhgMU78j+ctrYWQQBSUiMGZwgoxmWAB1PJgAC2QrgCIQ+EBsXxqGHuHzgY0yhxC2DgAEceEwkRMLFMZkhSnMQjY+EwOl15kR9GU4ABFHgQeBXEIwCwCH65f7IPyMmpMMr6ADCED403uUGgCJAPBsABVmek8TYAL4KoA)
```sh
--parser typescript
--print-width 120
```

**Input:**
```tsx
export const test = (x: string | number) =>
  typeof x === 'string'
	? parseInt(x) + 10
	: x + 20
```

**Output:**
```tsx
export const test = (x: string | number) => (typeof x === "string" ? parseInt(x) + 10 : x + 20);

```

**Expected behavior:**
I expected Prettier to only wrap code that extended beyond the print-width line; however it appears to be expanding code up to the maximum width allowed. This as side affect of ruining indentation that was intended to help readability of ternary operator

From some other issues on Prettier it seems it previous behavior was to preserve indentation of ternary:
https://github.com/prettier/prettier/issues/716

Did this change? Or is ternary indentation configurable?

**Comment by mattmazzola** [comment]
> No, the original ternary indentation has never been preserved. And it is not configurable.

Ok, it was incorrect expectations by me then.  This can be closed.  

After running prettier on our code base the ternaries were the only main difference (I realize now since most of them were well under the print-width) and I preferred the indentation form.  I thought I had configured it incorrectly and was causing the flattening.  I looked up on other issues and based on the other discussion mistakenly thought prettier did always indent them with leading `?` and `:`
